pkg/config/config.go

```diff
@@ -1,26 +1,75 @@
 package config
 
+import (
+	"fmt"
+
+	"github.com/caarlos0/env"
+	"github.com/golang/glog"
+)
+
 type DaemonConfig struct {
-	Plugin         string `json:"plugin"`          // Subnet manager plugin name
-	PeriodicUpdate int    `json:"periodic_update"` // Interval between every check for the added and deleted pods
-	GuidPool       GuidPoolConfig
-	SubnetManager  SubnetManagerPluginConfig
+	Plugin         string                    `json:"plugin" env:"PLUGIN"`                   // Subnet manager plugin name
+	PeriodicUpdate int                       `json:"periodic_update" env:"PERIODIC_UPDATE"` // Interval between every check for the added and deleted pods
+	GuidPool       GuidPoolConfig            `json:"guid_pool"`
+	SubnetManager  SubnetManagerPluginConfig `json:"subnet_manager"`
```

Comment:
need to remove the json annotation if not used

Reply:
removed